### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN020.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#66] [General]
# https://blog.3place.cn/3place/lmyn/LMYN020.html

https://blog.3place.cn/3place/lmyn/LMYN020.html

**Comment by picklecai** [comment]
父亲对儿子、年长者对年幼者的判断：罗摩太小，不能打败罗刹。我是父亲，我去，我拼尽全力。即使不成，那也是牺牲的我自己。我保护圈内的人，不能去冒这个险。

众友对罗摩的判断：他是上天注定的能打败罗刹的人，他是唯一一个。

其他路人：那个人，16岁，就要面临这样的选择。

罗摩本人：没见过，没试过。试试也不坏。

**Comment by picklecai** [comment]
罗刹头子罗波那，
力大无穷声名扬，
是财神爷的兄弟，
是毗尸罗婆的儿郎。 （1.19.17）

为什么强调财神爷的兄弟？？强调一个坏人很有钱？

**Comment by picklecai** [comment]
他还是一个孩子，婆罗门！
我怎么能把他交给你？ （1.19.23）

这是一个父亲的哀求。

我能够带着朋友们，
同其中的一个去比武。

这是一个父亲的无奈。
